src/components/slider/slider.scss

```diff
@@ -174,18 +207,11 @@ md-slider {
   /**
    * The border/background that comes in when focused in non-discrete mode
    */
-  .md-focus-thumb {
-    @include slider-thumb-position($slider-focus-thumb-width, $slider-focus-thumb-height);
-    display: none;
-    opacity: 0;
-    background-color: #C0C0C0;
-    animation: sliderFocusThumb 0.4s linear;
-  }
   .md-focus-ring {
     @include slider-thumb-position($slider-focus-thumb-width, $slider-focus-thumb-height);
-    transform: scale(0);
-    transition: all 0.2s linear;
-    opacity: 0.26;
+    transform: scale(.7);
+    opacity: 0;
+    transition: all ($slider-thumb-focus-duration / 2) $swift-ease-in-out-timing-function;
```

Comment:
When providing a custom duration, it helps to have a short comment as to why we don't use the default.